### PR TITLE
ADD: search bar

### PR DESCRIPTION
**Pull request by YuliiaMay**

### CODE REVIEW

File: src/components/common/SearchBar/SearchBar.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+
+const initialValues = {
+    search: '',
+};
+
+
+const SearchBar = () => {
+    const handleSubmit = ({search}, {resetForm}) => {
+        console.log(search);
```
Comment (maxpopovitch) [inline src/components/common/SearchBar/SearchBar.jsx]: Please remove it if it's not needed

File: src/components/common/SearchBar/SearchBar.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        resetForm();
```
Comment (maxpopovitch) [inline src/components/common/SearchBar/SearchBar.jsx]: Are you sure you need to reset the form on submit?

File: src/components/common/SearchBar/SearchBar.jsx (hunk 1 above)
Comment (YuliiaMay) [reply src/components/common/SearchBar/SearchBar.jsx]: has removed

File: src/components/common/SearchBar/SearchBar.jsx (hunk 2 above)
Comment (YuliiaMay) [reply src/components/common/SearchBar/SearchBar.jsx]: has removed

**Review by maxpopovitch — APPROVED** [review]

**Pull request merged**